R/write.R

```diff
@@ -141,34 +141,41 @@ detect.driver = function(filename) { #nocov start
 	ext <- tolower(tools::file_ext(filename))
 	if (nchar(ext) < 2) {
 		warning("file without extension, using driver GTiff", call. = FALSE)
-		"GTiff"
+		return("GTiff")
```

Comment:
Why use `return` here?

Reply:
For no good reason. I'll change that now and use single quotation marks for consistency.